web-js/search-widget-with-customized-geocoder-server/searchGeocodeServer.html

```diff
@@ -0,0 +1,152 @@
+   <title>ArcGIS API for JavaScript | Customize Search widget for Geocode Server</title>
+   <link rel="stylesheet" href="http://js.arcgis.com/3.13/dijit/themes/claro/claro.css">
+   <link rel="stylesheet" href="http://js.arcgis.com/3.13/esri/css/esri.css">
+   <style>
+      html,
+      body,
+      #map,
+      .map.container {
+         padding: 0;
+         margin: 0;
+         height: 100%;
+         width: 100%;
+      }
+      #info {
+         top: 2px;
+         color: #444;
+         height: auto;
+         font-family: arial;
+         font-weight: bold;
+         left: 69px;
+         margin: 5px;
+         padding: 10px;
+         position: absolute;
+         width: 260px;
+         z-index: 40;
+         border: solid 1px #003300;
+         border-radius: 4px;
+         background-color: #E5E5E5;
+      }
+      #search {
+         display: block;
+         position: absolute;
+         z-index: 2;
+         top: 85px;
+         left: 74px;
+      }
+      /*Beginning of search box modifications*/
+      .arcgisSearch .searchClear {
+         background-color: #E5E5E5;
+      }
+
+      .arcgisSearch .searchGroup .searchInput,
+      .arcgisSearch .searchBtn,
+      .arcgisSearch .noResultsMenu,
+      .arcgisSearch .suggestionsMenu {
+         border: 1px solid #003300;
+         background-color: #E5E5E5;
+      }
+      .arcgisSearch .noValueText {
+         color: red;
+         font-size: 14px;
+      }
+      /*Beginning of popup modifications*/
+      .esriPopup .titlePane {
+         background-color: #003300;
+         border-bottom: 1px solid #121310;
+         font-weight: bold;
+      }
+      .esriPopup a {
+         color: #DAE896;
+      }
+      .esriPopup .contentPane,
+      .esriPopup .actionsPane,
+      .esriPopup .pointer,
+      .esriPopup .outerPointer {
+         background-color: #B3B3B3;
+      }
+   </style>
+
+   <script>
+      var dojoConfig = {
+         parseOnLoad: true
+      };
+   </script>
+   <script src="http://js.arcgis.com/3.13/"></script>
+
+   <script>
+      var map;
+
+      require([
+      "esri/map",
+      "esri/layers/FeatureLayer",
+      "esri/dijit/Search",
+      "esri/tasks/locator",
+      "esri/symbols/PictureMarkerSymbol",
+      "esri/InfoTemplate",
+      "dojo/domReady!"
+      ], function (Map, FeatureLayer, Search, Locator, PictureMarkerSymbol, InfoTemplate) {
+
+         map = new Map("map", {
+            basemap: "dark-gray",
+            center: [-122.029133,47.521962], // lon, lat
+            zoom: 10
+         });
+         //ArcGIS Online feature service showing ecological footprints taken from Global FootPrint Network, 
+         //more information on this can be found http://jsapi.maps.arcgis.com/home/item.html?id=0f4c89208dce44b583d9219d843591c3
+         var layer = new FeatureLayer("http://services.arcgis.com/Ej0PsM5Aw677QF1W/arcgis/rest/services/KingCountySchoolDistricts/FeatureServer/0", {
```

Comment:
if you're trying to demo custom geocoding services, seems unhelpful to have a feature layer too...